openfisca_us/variables/gov/hhs/medicaid/geography/medicaid_rating_area.py

```diff
@@ -17,4 +17,18 @@ def formula(household, period, parameters):
             has_defined_mra, state, list(mra._children.keys())[0]
         )  # Fill in with any valid State to avoid errors
         three_digit_zip_code = household("three_digit_zip_code", period)
-        return where(has_defined_mra, mra[state][three_digit_zip_code], -1)
+        county_name = household("county_str", period)
+        mra_values = np.ones_like(state) * -1
+        unknown_location = (county_name == "UNKNOWN") & (three_digit_zip_code == "UNKNOWN")
+        for individual_state in mra._children.keys():
+            in_state = state == individual_state
```

Comment:
Can then use `target` instead in the places below
```suggestion
            target = (state == individual_state) & ~unknown_location
```